config/locales/en.yml

```diff
@@ -34,6 +34,7 @@ en:
     pattern_mismatch: Please enter information in the correct format
 
   links:
+    resend: Resend
     sign_out: 'Sign out'
     sign_in: 'Log in'
     sign_up: 'Sign up'
```

Comment:
Unrealted to this PR but why are these strings quotes and non of the others in this file are?